deployment/maintain/rebase-helper.md

```diff
@@ -20,29 +20,32 @@ These programs are installed automatically as dependencies of rebase-helper.
 
 Note: rebase-helper is also available as EPEL-7 package. Feel free to use it on CentOS and RHEL systems.
 
-## How does the rebase-helper work?
+## How does rebase-helper work?
 Rebase-helper workflow can be summarized in following steps:
 
 - Downloads an archive with new sources.
 - Rebases downstream patches on top of the latest upstream version using git rebase command.
-- If patching was successful, it builds two sets of RPMs. One using the ``old`` sources and a second one using the ``new`` sources. Available build tools are ``mock``, ``rpmbuild``, ``fedpkg``.
-  - If ``new`` RPMs build fails, downloads logs like build.log and root.log
+- If patching was successful, it builds two sets of RPMs. One using the **``old``** sources and a second one using the **``new``** sources. Available build tools are ``mock``, ``rpmbuild``, ``fedpkg``.
```

Comment:
Regarding the markdown changes, I wouldn't put both ``
and ** there.  Choose one, please.

Reply:
Well, AFAICT it's because of the syntax hilighting of key words, see resulting page:

https://developer.fedoraproject.org/deployment/maintain/rebase-helper.html

Thinking about it some more, maybe we could live the backtics instead

Reply:
That's the odd thing, though. `old` and `new` _aren't_ actually keywords, as far as I can tell. They're closer to argument placeholders, in fact I'm wondering if they shouldn't be italicized instead.